Source/src/components/ComponentAgent.h

```diff
@@ -48,6 +49,8 @@ namespace Hachiko
 		bool use_pathfinder = true;
 
 		bool show_debug_info = false;
+
+		bool player = false;
```

Comment:
Let's rename this to is_player, the name "player" feels like a non-boolean object.